client/my-sites/checkout/checkout/composite-checkout.js

```diff
@@ -275,6 +268,32 @@ function useRedirectIfCartEmpty( items, redirectUrl ) {
 	}, [ redirectUrl, items, prevItemsLength ] );
 }
 
+function useCountryList( overrideCountryList ) {
+	// Should we fetch the country list from global state?
+	const shouldFetchList = overrideCountryList?.length <= 0;
+
+	const [ countriesList, setCountriesList ] = useState( overrideCountryList );
+
+	const dispatch = useDispatch();
+	const globalCountryList = useSelector( state => getCountries( state, 'payments' ) );
+
+	// Has the global list been populated?
+	const listIsFetched = globalCountryList?.length > 0;
```

Comment:
Very minor, but I like to name booleans like this by starting with "is", so this could be changed to `isListFetched`.